extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -144,11 +146,7 @@ export class PublishDatabaseDialog {
 
 			// add SQLCMD variables table if the project has any
 			if (Object.keys(this.project.sqlCmdVariables).length > 0) {
-				this.formBuilder.insertFormItem({
-					title: constants.sqlCmdTableLabel,
-					component: <azdata.TableComponent>this.sqlCmdVariablesTable
-				},
-					6);
+				this.formBuilder.addFormItem(this.sqlCmdVariablesFormComponent, { titleFontSize: titleFontSize });
```

Comment:
changing this from insertFormItem to addFormItem because it's the table is now always the last thing (the load profile button used to be last, which is why the index was needed before)

Reply:
we cant have both? like kepe the index as well - or you sure its not needed?

Reply:
we could keep it, but it's harder to keep track of updating the index if more things are added so it's easier to remove it if the table is the last thing anyway